tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py35
+envlist = py3
```

Comment:
unintentional?

Reply:
Apologies for the wording in the title. This is intentional as I don't have python 3.5 installed on my system (I don't think many people have 3.5 installed, instead 3.7 or 3.8).

I could use python virtual envs, but changing the requirement from py35 to py3 just seemed more practical.

Reply:
Yeah, I'm in the same boat, change it to `py36` every time (and have been close to committing that change). 

The idea was to keep it the same as the CI version - perhaps the difference is not that great with newer python versions?